docs/FAQs/devtron-troubleshoot.md

```diff
@@ -96,4 +96,55 @@ https://grafana.com/api/dashboards/13320/revisions/4/download
 https://grafana.com/api/dashboards/13325/revisions/4/download
 https://grafana.com/api/dashboards/13321/revisions/6/download
 ```
-After that, your issue should be resolved and you should be able to see all the graphs on UI.
\ No newline at end of file
+After that, your issue should be resolved and you should be able to see all the graphs on UI.
+
+#### 6. Unable to Login Devtron dashboard even if the password is correct
+
+If you are not able to login your Devtron dashboard after giving the correct password, there can be high chance that the `argocd token` of previous session has been stored in the cookies and is not able to override the new token that is generated for the new session. If this is the issue you are facing, please follow the following steps.
```

Comment:
If you are not able to login into Devtron dashboard even after giving the correct password, it is possible that the `argocd token` of previous session has been stored in the cookies and is not able to override the new token that is generated for the new session.

Reply:
resolved